.github/workflows/labeler.yml

```diff
@@ -15,5 +15,7 @@ jobs:
       - uses: actions/labeler@v3
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
-          sync-labels: true
+          # Do not sync labels as this conflicts with manually added "tests:hive" label.
+          # Syncing labels requires that we define "components" labels.
```

Comment:
The config is so misnamed. It actually does "remove if reverted" - https://github.com/actions/labeler/pull/96/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R12.

Maybe simplify comment to 

```suggestion
          # Do not revert labels as this conflicts with manually added labels. The config name is misleading.
```

Reply:
`The config name is misleading` --- this is subjective, and not that important. We don't need to come across as derogatory. 
I will keep my current wording, if you don't mind?

Reply:
But `s/sync/revert` would help with understanding. Feel free to keep existing if you wish.

Reply:
```
# Do not sync labels as this reverts manually added "tests:hive" label.
```